modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/product_options/ProductOptionNumeric.js

```diff
@@ -58,7 +58,7 @@ const ProductOptionNumeric = ({
 			);
 
 			if (option) {
-				[value] = option.value;
+				value = option.value;
```

Comment:
Same here: is it correct we are removing the array?

Reply:
the value is always be a single value, only ProductOptionMultipleCheckbox supports multiple values. I am removing the value as array for consistency purposes along with the other changes I made for the other fields such as numeric and date.